docs/src/gs/externally-managed-infrastructure.md

```diff
@@ -32,6 +32,7 @@ spec:
   networkSpec:
     apiServerLoadBalancer:
       loadBalancerId: <OCID of Control Plane Endpoint LoadBalancer>
+    skipNetworkManagement: true
```

Comment:
I believe this is needed here as well.

Reply:
No, there are 2 cases, one case is when cluster.x-k8s.io/managed-by: "external" is used(the current doc) in which case all of infrastructure including apiserver lb is not reconciled.
The second case is when skipNetworkManagement is set to true(above annotation not present) in which case only apiserver lb is reconciled/managed/ I would suggest we move the section https://github.com/oracle/cluster-api-provider-oci/blob/main/docs/src/networking/custom-networking.md "Example spec for externally managed VCN infrastructure" to this doc, have 2 cases, externally managed infrastructure and externally managed network infrastructure and explain the 2 cases. All the content should be present in the docs already.